documentation/migration-guides/Az.1.0.0-migration-guide.md

```diff
@@ -0,0 +1,374 @@
+AzureRM.DataFactoryV2                 -> Az.DataFactory
+AzureRM.MachineLearningCompute        -> Az.MachineLearning
+AzureRM.Insights                      -> Az.Monitor
+AzureRM.RecoveryServices.Backup       -> Az.RecoveryServices
+AzureRM.RecoveryServices.SiteRecovery -> Az.RecoveryServices
+AzureRM.Tags                          -> Az.Resources
+Azure.Storage                         -> Az.Storage
+```
+
+The changes in module names mean that any script that uses ```#Requires``` or ```Import-Module``` to load specific modules will need to be changed to use the new module instead.
+
+#### Migrating #Requires Statements
+Scripts that use #Requires to declare a dependency on AzureRM modules should be updated to use the new module names
+```powershell
+#Requires -Module AzureRM.Compute
+```
+
+Should be changed to
+```powershell
+#Requires -Module Az.Compute
+```
+
+#### Migrating Import-Module Statements
+Scripts that use ```Import-Module``` to load AzureRM modules will need to be updated to reflect the new module names.
+```powershell
+Import-Module -Name AzureRM.Compute
+```
+
+Should be changed to
+```powershell
+Import-Module -Name Az.Compute
+```
+
+### Migrating Fully-Qualified Cmdlet Invocations
+Scripts that use module-qualified cmdlet invocations, like
+```powershell
+AzureRM.Compute\Get-AzureRmVM
+```
+
+Should be changed to use the new module and cmdlet names
+```powershell
+Az.Compute\Get-AzVM
+```
+
+### Migrating Module Manifest Dependencies
+Modules that express dependencies on AzureRM modules through a module manifest (.psd1) file will need to updated the module names in their 'RequiredModules' section
+
+```powershell
+RequiredModules = @(@{ModuleName="AzureRM.Profile"; ModuleVersion="5.8.2"})
+```
+
+Should be changed to
+
+```powershell
+RequiredModules = @(@{ModuleName="Az.Profile"; ModuleVersion="1.0.0"})
+```
+
+### Removed modules
+- `AzureRM.Backup`
+- `AzureRM.Compute.ManagedService`
+- `AzureRM.Scheduler`
+
+The tooling for these services are no longer actively supported.  Customers are encouraged to move to alternative services as soon as it is convenient.
+
+### Windows PowerShell 5.1 and .NET 4.7.2
+- Using Az with Windows PowerShell 5.1 requires the installation of .NET 4.7.2. However, using Az with PowerShell Core does not require .NET 4.7.2. 
+
+### Temporary removal of User login using PSCredential
+- Due to changes in the authentication flow for .NET Standard, we are temporarily removing user login via PSCredential. This capability will be re-introduced in the 1/15/2019 release for WIndows PowerShell 5.1. 
+
+### Default Device Code login instead of Web Browser prompt
+- Due to changes in the authentication flow for .NET Standard, we are using device login as the default login flow during interactive login. Web browser based login will be re-introduced for WIndows PowerShell 5.1 as the default in the 1/15/2019 release. At that time, device login will available via a parameter.
```

Comment:
@markcowl "...device login will available..." --> "...device login will _be_ available..."

Also, should we mention that we will default to device code login if anything goes wrong with user login?